.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,9 @@
 <!--
-Please complete the following sections when you submit your pull request.
-You are encouraged to keep this top level comment box updated as you
-develop and respond to reviews.
-Note that text within html comment tags will not be rendered.
+Please complete the following sections when you submit your pull request. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. Note that text within html comment tags will not be rendered.
 -->
 ### Summary
 
-<!-- Describe the problem you're trying to fix in this pull request. Please
-reference any related issue and use fixes/close to automatically close them,
-if pertinent -->
+<!-- Describe the problem you're trying to fix in this pull request. Please reference any related issue and use fixes/close to automatically close them, if pertinent -->
 
 Fixes #<NUM> <!-- (e.g. Fixes #58.) -->
```

Comment:
My personal opinion is that I don't like this comment after the line and would prefer it to be above so it's a bit clearer that it's something I have to change or delete if it's not relevant. Minor thing, but it does bug me every time I open an issue on the repo.

Reply:
```suggestion
 <!-- (e.g. Fixes #58.) -->
Fixes #<NUM>
```
Something like this @sgibson91?

Reply:
Yes please! Or perhaps:
```
<!-- You can automatically close issues with pull requests like so: Fixes #58 -->
```